src/pages/community/how-to-contribute.md

```diff
@@ -1,12 +1,122 @@
+Apache Ozone welcomes contributions in many forms, not just code. Here are various ways you can contribute:
+
+### 1. Code Contributions
+
+#### Finding Issues to Work On
+
+- **Newbie Issues**: Start with [beginner-friendly issues](https://issues.apache.org/jira/issues/?filter=12353868) that are good entry points
+- **All Open Issues**: Browse [unassigned Jira issues](https://s.apache.org/OzoneUnassignedJiras) to find something that matches your interests and skills
+
+#### Making Code Changes
+
+1. **Set Up Development Environment**
+   - Fork and clone the repository
+   - Enable the `build-branch` GitHub Actions workflow in your fork
+
+2. **Create a Branch**
+   - Ensure a Jira issue exists for your task (e.g., HDDS-1234)
+   - Create a branch for your work: `git checkout -b HDDS-1234`
+
+3. **Develop Your Changes**
+   - Follow code style guidelines (2 spaces indentation, 120-char line length)
+   - Write tests for your changes
+   - Run relevant checks from `hadoop-ozone/dev-support/checks/` directory
+
+4. **Submit Your Contribution**
+   - Push changes to your fork
+   - Wait for the `build-branch` workflow to complete
+   - Create a pull request with a clear description
+   - Update the Jira issue status to "Patch Available"
+
+### 2. Documentation
+
+- **Website Improvements**: Contribute to the [Ozone website](https://ozone.apache.org/) - see [instructions](https://cwiki.apache.org/confluence/display/OZONE/Modifying+the+Ozone+Website)
+- **Developer Docs**: Enhance the [markdown documentation](https://github.com/apache/ozone/tree/master/hadoop-hdds/docs/content) in the source tree
+- **Wiki Pages**: Update the [project wiki](https://cwiki.apache.org/confluence/display/OZONE) (request access through dev@ozone.apache.org)
+
+### 3. Testing
+
+- **Unit Tests**: Write JUnit tests for Java code
+- **Acceptance Tests**: Create Docker + Robot Framework tests
+- **Blockade Tests**: Develop Python + Blockade tests
+- **Performance Testing**: Run benchmarks with `ozone freon` and report findings
+
+### 4. Visual Design & Art
+
+Ozone welcomes contributions in design and visual arts:
+
+- **Logos and Icons**: Create variations of the Ozone logo for different contexts
+- **Diagrams**: Design architectural and conceptual diagrams for documentation
+- **Website Design**: Improve the look and feel of the Ozone website
+- **Slide Decks**: Design templates for presentations about Ozone
+
+Share your artwork through the [GitHub Discussions](https://github.com/apache/ozone/discussions) or on the dev mailing list dev@ozone.apache.org.
+
+### 5. Bug Reports and Feature Requests
+
+- Report bugs or suggest features through [Jira](https://issues.apache.org/jira/projects/HDDS/)
+- Search existing issues before creating a new one
+- Provide detailed steps to reproduce bugs
+
+### 6. Community Support
+
+- Answer questions on [GitHub Discussions](https://github.com/apache/ozone/discussions)
+- Help review pull requests
+- Join community calls and discussions
```

Comment:
We can link to the [communication channels](https://ozone-site-v2.staged.apache.org/community/communication-channels) page here for more info.